submissions/IhorOzerov/a-tiny-JS-world/index.js

```diff
@@ -0,0 +1,62 @@
+const man = {
+   creature: 'human',
+   name: "Gregory",
+   gender: 'male',
+   hands: 2,
+   legs: 2,
+   say: "Hello, woman!",
+   friends: ["Bandit", "Gangster"]
+};
+const woman = {
+   creature: 'human',
+   name: "Yohanna",
+   gender: 'female',
+   hands: 2,
+   legs: 2,
+   say: "Hello, man!",
+   friends:"Gangster"
+};
+const dog = {
+   creature: 'dog',
+   name: "Bandit",
+   gender: 'male',
+   hands: 0,
+   legs: 4,
+   say: "Bark!",
+   friends:  ["Gregory", "Yohanna"]
+};
+const cat = {
+   creature: 'cat',
+   name: "Gangster",
+   gender: 'male',
+   hands: 0,
+   legs: 4,
+   say: "Where is my money, Gregory?!",
+   friends:  ["Gregory", "Vanessa"]
+};
+const catWoman = Object.create(cat);
+catWoman.creature = "cat-woman";
+catWoman.name = "Vanessa";
+catWoman.gender = 'female';
+catWoman.legs = 2;
+catWoman.hands = 2;
+catWoman.friends =  ["Yohanna", "Gangster"];
```

Comment:
You do not need to fix anything here. Just to reflect on when you re-approach TJSW after a while (OOP Exercise).
Would it be more efficient to extend CatWoman from Woman (or Human) and hard-bind only saying with Cat?